dataProcessing/uploading_clinical_summary.Rmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "Pushing latest sample summary to synapse"
+author: "Camilo Posso"
+date: "10/28/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(openxlsx)
+source("../util/synapseUtil.R")
+```
+
+
+
+We just received the latest sample summary table, containing a lot of annotation
+data, including allele ratios. 
+
+
+
+```{r loading data}
+path <- "../proteomics/data/BeatAML_complete_sample_summary/PNNL_clinical_summary_12_08_2021.xlsx"
```

Comment:
where is this data? it should not be on the github repo...

Reply:
It's not on github, it was file in my local folders. This "Data" folder folder is excluded from this repo, I am just uploading this to synapse with this script. See here for the file:
https://www.synapse.org/#!Synapse:syn26532699

Reply:
Ok, then please replace with a syn get. all code should be runnable for any of us with access.

Reply:
For clarity, uploading_clinical_summary.Rmd is there for document the upload of a new table summarizing clinical data for the patients to synapse. It's not meant to be run multiple times by someone with synapse access, as it needs the local file to push to synapse. I'll try to add a small comment to these kinds of scripts mentioning this from now.